### PR TITLE
pari, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: sci-mathematics/pari/patches/pari-2.17.0.patchset [hunk 1]
```diff
@@ -1,18 +1,21 @@
-From eb88d504cbfea312e6d65ab89e48eea43fa560ac Mon Sep 17 00:00:00 2001
+From 23cec188856bbab6246998fd0f36ecc5ee90a576 Mon Sep 17 00:00:00 2001
 From: Jerome Duval <jerome.duval@gmail.com>
 Date: Sat, 10 May 2014 12:15:25 +0000
 Subject: we only use release versions, no point in checking .git
 
 
+diff --git a/config/arch-osname b/config/arch-osname
+old mode 100644
+new mode 100755
```
Comment (jmairboeck) [inline sci-mathematics/pari/patches/pari-2.17.0.patchset:9]: Did you intentionally mark this file as executable?

Comment (Begasus) [reply sci-mathematics/pari/patches/pari-2.17.0.patchset:9]: err no, wondered about that :D probably why it failed to apply the first time ...

Comment (Begasus) [reply sci-mathematics/pari/patches/pari-2.17.0.patchset:9]: Somehow haikuporter changed something here? I don't know, but the mentioned file in the source "is" executable (bash script), so it's not wrong in this context?

Comment (jmairboeck) [reply sci-mathematics/pari/patches/pari-2.17.0.patchset:9]: OK, maybe git or haikuporter did something itself here.

It is probably no problem, as long as the patchset still applies cleanly.

**Pull request merged**